pkg/network/link/mac.go

```diff
@@ -19,8 +19,29 @@
 
 package link
 
+import (
+	"fmt"
+	"net"
+)
+
 const StaticMasqueradeBridgeMAC = "02:00:00:00:00:00"
 
 func IsReserved(mac string) bool {
 	return mac == StaticMasqueradeBridgeMAC
 }
+
+// ValidateMacAddress performs a validation of the address validity in terms of format and size.
+// An empty mac address input is ignored (i.e. is considered valid).
+func ValidateMacAddress(macAddress string) error {
+	if macAddress != "" {
```

Comment:
Please consider reversing the condition in order to return early and decrease indentation.

Reply:
I think this is classified as a nit, but it does not worth debating on it any further.
Done.